pom.xml

```diff
@@ -162,4 +164,59 @@
       </pluginRepository>
     </pluginRepositories>
 
+    <profiles>
+        <profile>
+            <id>license</id>
+            <build>
+                <defaultGoal>license:format</defaultGoal>
+                <plugins>
+                    <plugin>
+                        <groupId>com.mycila</groupId>
+                        <artifactId>license-maven-plugin</artifactId>
+                        <version>${mycila-license.version}</version>
+                        <configuration>
```

Comment:
I propose to add `<failIfUnknown>true</failIfUnknown>` or otherwise the plugin will ignore the file types it does not know. See http://code.mycila.com/license-maven-plugin/reports/3.0/check-mojo.html#failIfUnknown

Reply:
done